listable-demo/staff/views.py

```diff
@@ -85,3 +85,7 @@ def get_extra(self):
          """.format(cta.pk, ctb.pk)
 
         return {"select": {'genericname': extraq}}
+
+
+class StaffListWithLiveFilters(StaffList):
```

Comment:
I think this isn't working in the demo app because it's using the same template as the `StaffList` view which uses `{% listable 'staff-list' save_state=True pagination_type="bootstrap3" css_input_class="input-xs " %}`

Reply:
Added a custom template for the live filters view.